samples/boards/nrf52/mesh/onoff-app/README.rst

```diff
@@ -0,0 +1,87 @@
+.. _bluetooth-mesh-onoff-sample:
+
+Bluetooth: Mesh OnOff Model
+###########################
+
+Overview
+********
+
+This is a simple application demonstrating a Bluetooth mesh multi-element node. Each element has a mesh onoff client and server
+model which controls one of the 4 sets of buttons and LEDs .
+
+Prior to provisioning, the board broadcasts an unprovisioned beacon containing a unique UUID. It is
+obtained from the device address set by Nordic in the FICR. Each button inverts the state of its
+corresponding LED and does not initiate any mesh activity.
+
+The models for button 1 and LED 1 are in the node's root element.
+The 3 remaining button/LED pairs are in elements 1 through 3.
+Assuming the provisioner assigns 0x100 to the root element,
+the secondary elements will appear at 0x101, 0x102 and 0x103.
+
+It's anticipated that after provisioning, the button clients would
+be configured to publish and the LED servers to subscribe.
+
+If a LED server is provided with a publish address, it will
+also publish its status on an onoff state change.
+
+Messages from a button to its corresponding LED are ignored as
+the LED's state has already been changed locally by the button client.
+
+
+Requirements
+************
+
+* This sample is specific to the Nordic nRF52840-PDK board.
+
+Building and Running
+********************
+
+This sample can be found under :file:`samples/boards/nrf52/mesh/onoff-app` in the
+Zephyr tree.
+
+The following commands build the application.
+
+.. code-block:: console
+
+   # cd samples/boards/nrf52/mesh/onoff-app
+   # mkdir build
+   # cd build
+   # cmake -DBOARD=nrf52840_pca10056 ..
+   # make
+   # make flash
+
+Provisioning is done using BlueZ' meshctl utility. Below is an example that binds
```

Comment:
to avoid the possessive apostrophe, which would need an "s" after it, how about instead:

    using the BlueZ meshctl utility.

Reply:
Fixed thanks